src/Promitor.Tests.Unit/Validation/Scraper/Metrics/ResourceTypes/ExpressRouteCircuitsMetricsDeclarationValidationStepsTests.cs

```diff
@@ -0,0 +1,132 @@
+using System.ComponentModel;
+using Microsoft.Extensions.Logging.Abstractions;
+using Promitor.Agents.Scraper.Validation.Steps;
+using Promitor.Tests.Unit.Builders.Metrics.v1;
+using Promitor.Tests.Unit.Stubs;
+using Xunit;
+
+namespace Promitor.Tests.Unit.Validation.Scraper.Metrics.ResourceTypes
+{
+    [Category("Unit")]
+    public class ExpressRouteCircuitsMetricsDeclarationValidationStepTests : MetricsDeclarationValidationStepsTests
+    {
+        [Fact]
+        public void ExpressRouteCircuitsMetricsDeclaration_DeclarationWithoutAzureMetricName_Fails()
+        {
+            // Arrange
+            var rawDeclaration = MetricsDeclarationBuilder.WithMetadata()
+                .WithExpressRouteCircuitsMetric(azureMetricName: string.Empty)
+                .Build(Mapper);
+            var metricsDeclarationProvider = new MetricsDeclarationProviderStub(rawDeclaration, Mapper);
+
+            // Act
+            var scrapingScheduleValidationStep = new MetricsDeclarationValidationStep(metricsDeclarationProvider, NullLogger<MetricsDeclarationValidationStep>.Instance);
+            var validationResult = scrapingScheduleValidationStep.Run();
+
+            // Assert
+            Assert.False(validationResult.IsSuccessful, "Validation is successful");
```

Comment:
Yesterday I've introduced new asserts, would you mind using those please? https://github.com/tomkerkhove/promitor/pull/1252

You can use `PromitorAssert.ValidationFailed(validationResult);` & `PromitorAssert.ValidationIsSuccessful(validationResult);`